jetbrains: 2023.2 -> 2023.2.1

dritter opened the pull request:
## Description of changes

jetbrains.clion: 2023.2 -> 2023.2.1
jetbrains.datagrip: 2023.2 -> 2023.2.1
jetbrains.dataspell: 2023.2 -> 2023.2.1
jetbrains.gateway: 2023.2 -> 2023.2.1
jetbrains.goland: 2023.2 -> 2023.2.1
jetbrains.idea-community: 2023.2 -> 2023.2.1
jetbrains.idea-ultimate: 2023.2 -> 2023.2.1
jetbrains.phpstorm: 2023.2 -> 2023.2.1
jetbrains.pycharm-community: 2023.2 -> 2023.2.1
jetbrains.pycharm-professional: 2023.2 -> 2023.2.1
jetbrains.rider: 2023.2 -> 2023.2.1
jetbrains.ruby-mine: 2023.2 -> 2023.2.1
jetbrains.webstorm: 2023.2 -> 2023.2.1

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

dritter commented:
Result of `nixpkgs-review pr 251037` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.idea-community</li>
    <li>jetbrains.idea-ultimate</li>
  </ul>
</details>

dritter commented:
Result of `nixpkgs-review pr 251037` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>10 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.dataspell</li>
    <li>jetbrains.gateway</li>
    <li>jetbrains.idea-community</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.pycharm-community</li>
    <li>jetbrains.pycharm-professional</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

dritter commented:
Result of `nixpkgs-review pr 251037` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>11 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.dataspell</li>
    <li>jetbrains.gateway</li>
    <li>jetbrains.idea-community</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.pycharm-community</li>
    <li>jetbrains.pycharm-professional</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

niknetniko reviewed (approved):
`jetbrains.idea-ultimate` works for me

dritter committed:
jetbrains: 2023.2 -> 2023.2.1

jetbrains.clion: 2023.2 -> 2023.2.1
jetbrains.datagrip: 2023.2 -> 2023.2.1
jetbrains.dataspell: 2023.2 -> 2023.2.1
jetbrains.gateway: 2023.2 -> 2023.2.1
jetbrains.goland: 2023.2 -> 2023.2.1
jetbrains.idea-community: 2023.2 -> 2023.2.1
jetbrains.idea-ultimate: 2023.2 -> 2023.2.1
jetbrains.phpstorm: 2023.2 -> 2023.2.1
jetbrains.pycharm-community: 2023.2 -> 2023.2.1
jetbrains.pycharm-professional: 2023.2 -> 2023.2.1
jetbrains.rider: 2023.2 -> 2023.2.1
jetbrains.ruby-mine: 2023.2 -> 2023.2.1
jetbrains.webstorm: 2023.2 -> 2023.2.1

dritter committed:
jetbrains.plugins: update

dritter commented:
Result of `nixpkgs-review pr 251037` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>13 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.dataspell</li>
    <li>jetbrains.gateway</li>
    <li>jetbrains.goland</li>
    <li>jetbrains.idea-community</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.pycharm-community</li>
    <li>jetbrains.pycharm-professional</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

yrd reviewed (approved):
Tested Gateway and (remote) IDEA Ultimate, both seem to work 👍🏻

MartinRamm commented:
Tested `jetbrains.webstorm` on `x86_64`, works without any new problems (I can still reproduce #247148 , but that has nothing to do with the changes here)
